py/server/deephaven/_table_reader.py

```diff
@@ -67,7 +67,6 @@ def _table_reader_chunks(table: Table, *, cols: Optional[Union[str, Sequence[str
 
     row_sequence_iterator = row_set.getRowSequenceIterator()
     col_sources = [table.j_table.getColumnSource(col_def.name) for col_def in col_defs]
-    chunk_size = row_set.size() if not chunk_size else chunk_size
```

Comment:
It looks like `chunk_size` could still be `None` (at least with current type hints); in that case, I'm assuming call to `_JTableUpdateDataReader.makeContext` would break?